torch/_dynamo/config.py

```diff
@@ -487,9 +484,9 @@ def default_debug_dir_root():
     "data",
 ]
 
-_autograd_backward_strict_mode_banned_ops.extend(
-    [name for name, _ in inspect.getmembers(torch.Tensor) if re.match(r"^is_.*", name)]
-)
+# _autograd_backward_strict_mode_banned_ops.extend(
+#     [name for name, _ in inspect.getmembers(torch.Tensor) if re.match(r"^is_.*", name)]
+# )
 
```

Comment:
Not sure if it's safe to remove this completely.

Reply:
It depends on if the autograd engine guarantees this. There is code somewhere where autograd requires the gradients to have the same shape and dtype (but I am not sure if this is true for all the other properties). So, we should go in and audit each property and check if autograd guarantees that the condition is known.

Reply:
This is the list of all attributes/functions starting with `is_` of `torch.Tensor`:
```
['is_coalesced', 'is_complex', 'is_conj', 'is_contiguous', 'is_cpu', 'is_cuda', 'is_distributed', 'is_floating_point', 'is_inference', 'is_ipu', 'is_leaf', 'is_maia', 'is_meta', 'is_mkldnn', 'is_mps', 'is_mtia', 'is_neg', 'is_nested', 'is_nonzero', 'is_pinned', 'is_quantized', 'is_same_size', 'is_set_to', 'is_shared', 'is_signed', 'is_sparse', 'is_sparse_csr', 'is_vulkan', 'is_xla', 'is_xpu']
```
I did a basic audit and updated the banned list following this change.